integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseOpenUtils.java

```diff
@@ -11,63 +11,13 @@
 
 package alluxio.fuse;
 
-import jnr.constants.platform.OpenFlags;
+import static jnr.constants.platform.OpenFlags.*;
```

Comment:
Alluxio code convention including not allowed *, all the import needs to be inline. Appreciate